docs/aca/01-deploy-api-to-aca/index.md

```diff
@@ -73,7 +73,7 @@ In this module, we will start by creating the first microservice named `ACA Web
     ```
 - From VS Code Terminal tab, open developer command prompt or PowerShell terminal and navigate to the parent directory which hosts the `.csproj` project folder and build the project. 
     ```shell
-    cd {YourLocalPath}\TasksTracker.TasksManager.Backend.Api
+    cd {YourLocalPath}\TasksTracker.ContainerApps\TasksManager.Backend.Api
```

Comment:
I believe instead of updating these paths, what actually needs to be fixed is this command here 

[TasksTracker](https://github.com/Azure/aca-dotnet-workshop/blob/main/docs/aca/01-deploy-api-to-aca/index.md?plain=1#L20)

Instead of 

 ```shell
  dotnet new webapi -o TasksManager.Backend.Api
  ```

It should have been 

```shell
  dotnet new webapi -o TasksTracker.TasksManager.Backend.Api
 ```

Reply:
That works - partially. Still throughout, there should be consistency since module 1: step 1 says to create the TasksTracker.ContainerApps folder. 

So, anytime a subdirectory is referenced, I believe it would be clearest to be consistent and use {YourLocalPath}\TasksTracker.ContainerApps\...

If not, I would suggest pulling out step 1.

Reply:
Yes, that I can agree with.

Reply:
Can you please update the PR ?

Reply:
Note: 03-aca-dapr-integration changes to use the tilde. My $.02... pick a way to reference dirs and stick with it throughout. :)

`
~\TasksTracker.ContainerApps\TasksTracker.WebPortal.Frontend.Ui> dapr run --app-id tasksmanager-frontend-webapp --app-port <web frontend application https port found under properties->launchSettings.json. e.g. 7000> --dapr-http-port 3501 --app-ssl -- dotnet run --launch-profile https
`

Reply:
Yup I agree. Easier to copy paste as well. 

This would mean we start here https://github.com/Azure/aca-dotnet-workshop/blob/main/docs/aca/01-deploy-api-to-aca/index.md?plain=1#L13

with as well. Agree?

````
mkdir ~/TasksTracker.ContainerApps
````

Reply:
Since you have already opened the PR, If you can already update module 1 and 2.. and then probably we can revisit entire workshop to make it consistent or will be glad if you want to do that part as well. 

Thanks again for spotting this and suggestions

Reply:
Yup, I can make the changes to this PR and throughout my journey in the workshop. Just to be clear, the preference is to use "TasksTracker.TasksManager.Backend.Api" for the API and ~\TasksTracker.ContainerApps\ for referencing the parent dir.

Reply:
Sounds great